modules/web-console/frontend/app/components/list-of-registered-users/column-defs.ts

```diff
@@ -24,7 +24,7 @@ const MODEL_HEADER_TEMPLATE = `<div class='ui-grid-cell-contents' bs-tooltip dat
 const CACHE_HEADER_TEMPLATE = `<div class='ui-grid-cell-contents' bs-tooltip data-title='{{ col.headerTooltip(col) }}' data-placement='top'><i class='fa fa-database'></i>${ICON_SORT}</div>`;
 
 const EMAIL_TEMPLATE = '<div class="ui-grid-cell-contents"><a bs-tooltip data-title="{{ COL_FIELD }}" ng-href="mailto:{{ COL_FIELD }}">{{ COL_FIELD }}</a></div>';
-const DATE_WITH_TITLE = '<div class="ui-grid-cell-contents"><label bs-tooltip data-title="{{ COL_FIELD | date:\'M/d/yy HH:mm\' }}">{{ COL_FIELD | date:"M/d/yy HH:mm" }}</label></div>';
+const DATE_WITH_TITLE = '<div class="ui-grid-cell-contents"><label bs-tooltip data-title="{{ COL_FIELD === -1 ? \'n/a\' : COL_FIELD | date:\'M/d/yy HH:mm\' }}">{{ COL_FIELD === -1 ? \'n/a\' : COL_FIELD | date:"M/d/yy HH:mm" }}</label></div>';
```

Comment:
1. Move "n/a" to locale file (messages.en.json) under `admin.listOfRegisteredUsers.noData.valueNotAvailable` path.
2. Use i18n filter to display the value, i.e. `COL_FIELD === -1 ? 'admin.listOfRegisteredUsers.noData.valueNotAvailable'|translate : COL_FIELD | date:\'M/d/yy HH:mm\'`.

Reply:
Refactored to use translation.